Enable/Utilities/ErrorHandler.m

```diff
@@ -48,4 +51,32 @@ + (void)testInternetConnection: (UIViewController* _Nonnull)vc {
 
     [internetReachable startNotifier];
 }
+
+
+
+#pragma mark - Loading
++ (void) startLoading: (UIView * _Nonnull) view {
```

Comment:
It doesn't make sense for me to have this in the error handler, a loading state is a view controller state, it's not an error, this should be in your view controllers. I appreciate the thought of not duplicating code, but you need to find another way of doing this, not in the error handler.